packages/frontend/src/components/ProjectConnection/DbtSettingsForm.tsx

```diff
@@ -31,12 +31,14 @@ interface DbtSettingsFormProps {
     disabled: boolean;
     defaultType?: DbtProjectType;
     selectedWarehouse?: SelectedWarehouse | undefined;
+    methods: UseFormReturn<any, object>;
```

Comment:
You can get the `reset` method from `useFormContext` and avoid prop drilling. 
https://react-hook-form.com/api/useformcontext